tron/core/action.py

```diff
@@ -55,11 +60,45 @@ def copy(self):
         return ActionCommandConfig(**self.state_data)
 
     @staticmethod
-    def to_json(state_data: dict) -> Optional[str]:
+    def from_json(state_data: str):
+        """Deserialize a JSON string to an ActionCommandConfig instance."""
+        # Here we will also need to load the json for "command_config" since it's a string
```

Comment:
oh it's under the try block 
```
json_data = json.loads(state_data)
```

Reply:
Maybe I should move the comment inside the try block

Reply:
oh, I see - i thought this was referring ot needing to json.loads something inside that string :p

imo, the type hint for state_data being a string is probably enough - especially since we don't have the same comment in the other from_json functions